index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+    <link rel="stylesheet" href="./css/style.css">
+    
+    <title>Document</title>
+</head>
+<body>
+    <header class="nav">
+        <a class="logo" href="#"><b>My Lgo</b></a>
+        <a href="#"><i class="material-icons" >	list</i></a>
+    </header>
+    <section class="container">
+    
+        <div class="content">
+            <h1><b>I’m Alexandra.<br>
+
+                Glad to see<br>
+                you!</b></h1>
+            <p>I’m a software developer! I can help youbuild a product , feature or website Lookthrough some of my work and
+                experience! If you like what you see and
+                have a project you need coded, don’t
+                hesitate to contact me.<br>
+                <span>LETS CONNECT</span></p>
+        <div class="icons">
+            <a class="icon1" href="#"><img class="icon1" src="Vector (1).jpg" alt=""></a>
+            <a class="icon2" href="#"><img class="icon2" src="Vector (2).jpg" alt=""></a>
+            <a class="icon3" href="#"><img class="icon3" src="Vector (3).jpg" alt=""></a>
+            <a class="icon4" href="#"><img class="icon4" src="Vector (4).jpg" alt=""></a>
+            <a class="icon5" href="#"><img class="icon5" src="Vector (5).jpg" alt=""></a>
+            <a class="icon6" href="#"><img class="icon6" src="Vector (6).jpg" alt=""></a>
+            <a class="icon7" href="#"><img class="icon7" src="Vector (7).jpg" alt=""></a>
+        </div>    
```

Comment:
- [ ] As requested by the previous reviewer kindly consider making sure that the list of icons uses a ul tag to group them and li per icon; each icon has a link (a tag). Apart from being semantically correct, the tag will let you connect your social networks with the icons, using href ⛏